run_fal.py

```diff
@@ -40,6 +41,7 @@
 parser.add_argument("--team-stats", action="store_true")
 parser.add_argument("--team-dist", action="store_true")
 parser.add_argument("--team-score", action="store_true")
+parser.add_argument("--bench-swap", action="store_true")
```

Comment:
Should we update the README by adding `--bench-swap` to the instructions?

Reply:
Yeah, was going to do that before promoting this from draft 😀